src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -108,15 +99,9 @@ public function getQueryBuilderPartsForCachingHash($queryBuilder)
         return false;
     }
 
-    public function __construct(ManagerRegistry $registry, PropertyAccessorInterface $propertyAccessor = null, ChoiceListFactoryInterface $choiceListFactory = null)
+    public function __construct(ManagerRegistry $registry)
```

Comment:
This is not a BC break because PHP does not care about additional arguments that you pass to the constructor/method. Check this http://symfony.com/doc/current/contributing/code/bc.html#id13